### PR TITLE
update go version to 1.18.9

### PR DESCRIPTION
**Pull request by dhiaayachi**
### Description
update go version to `1.18.9`

### CODE REVIEW

**Comment by eculver** [comment]
We don't need the `backport/1.13.x` label since this is going directly against `release/1.13.x`.

**Comment by dhiaayachi** [comment]
> We don't need the `backport/1.13.x` label since this is going directly against `release/1.13.x`.

right 😆 !
I'm hitting a bump with this one. Something wrong with the ci image?

File: .circleci/config.yml [hunk 1]
```diff
@@ -208,6 +208,8 @@ jobs:
       - checkout
       - run: go env
       - run:
+          environment: 
+           GOARCH: ""
           name: Install golangci-lint
```
Comment (rboyer) [inline .circleci/config.yml:213]: After this targeted fix we should get this on main etc.

**Review by rboyer — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (dhiaayachi) [reply .circleci/config.yml:213]: yes

**Pull request merged**